examples/log-to-file.md

```diff
@@ -0,0 +1,72 @@
+# Log to File
+
+- [Basic Example](#basic-example)
+- [Separators in JSON Streams](#Separators-in-JSON-streams)
+- [Log Rotation](#log-rotation)
+
+## Basic Example
+
+```
+file: [{
+    module: 'good-squeeze',
+    name: 'Squeeze',
+    args: [{ ops: '*' }]
+}, {
+    module: 'good-squeeze',
+    name: 'SafeJson'
+}, {
+    module: 'good-file',
+    args: ['./test/fixtures/awesome_log']
+}]
+```
+
+This reporter spec logs ops events to a file.
+
+It uses the `good-squeeze` Squeeze function to select only ops events.
+
+Events that pass the filter stream to `good-squeeze` SafeJson for formatting.
+
+JSON-formatted events stream to `good-file`, which writes them to `./test/fixtures/awesome_log`.
+
+### Separators in JSON Streams
+
+To insert a comma between JSON-formatted objects, add `args` to the SafeJson formatter. `good` passes the `args` array to the [SafeJson constructor](https://github.com/hapijs/good-squeeze#safejsonoptions-stringify).
+
+```
+file: [{
+    module: 'good-squeeze',
+    name: 'Squeeze',
+    args: [{ ops: '*' }]
+}, {
+    module: 'good-squeeze',
+    name: 'SafeJson',
+    args: [{}, { separator: ',' }]
```

Comment:
I think this can be `null` rather than en empty object. Now that I see this, I really messed up the arguments to the constructor... opps.